packages/terra-list/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Fixed
+* Chevron positioning
```

Comment:
More details on this fix would be helpful if we every need to modify this in the future. Like what browsers it was an issue in, etc.

Reply:
could include that you applied it's using flexbox now as well